independent-projects/arc/processor/src/main/java/io/quarkus/arc/processor/ObserverGenerator.java

```diff
@@ -258,6 +261,12 @@ protected void initMaps(ObserverInfo observer, Map<InjectionPointInfo, String> i
         }
     }
 
+    protected void implementIfExistsGetReception(ClassCreator observerCreator) {
+        MethodCreator getObservedType = observerCreator.getMethodCreator("getReception", Reception.class)
+                .setModifiers(ACC_PUBLIC);
+        getObservedType.returnValue(getObservedType.load(Reception.IF_EXISTS));
```

Comment:
Careless copy'n'paste detected! :laughing:

```suggestion
        MethodCreator getReception = observerCreator.getMethodCreator("getReception", Reception.class)
                .setModifiers(ACC_PUBLIC);
        getReception.returnValue(getReception.load(Reception.IF_EXISTS));
```

Reply:
Guilty as charged!
Should be fixed now :)